frontend/src/components/LandingPage/Team/Team.js

```diff
@@ -0,0 +1,77 @@
+/*
+This component will have a title and description of the team members.
+*/
+
+import React from 'react';
+import { Card, Button, CardTitle, CardText, CardSubtitle, CardDeck} from 'reactstrap';
+import './Team.css'
+
+document.body.style.backgroundColor="#e3c3b2";
+
+function Team(){
+    return (
+        <div className ="container">
+        <h1 className = "home">Meet the Team!</h1>
```

Comment:
What do you think of changing this h1 className from just "home" to "home display-4" just to keep that header font consistent through the landing page?

Reply:
For sure!